ksy_style_guide.adoc

```diff
@@ -249,28 +248,36 @@ guesswork.
 
 * For simple non-repeated fields, use a simple singular form —
   i.e. `width`, `header`, `transaction_id`, `file`.
-* For repeated files (i.e. with `repeat: something`), use plular form
+* For a sequence (i.e. with `repeat: something`), use plular form
   — i.e. `files`, `transactions`.
 * Don't be overly verbose: use commonly understood abbreviations
   liberally, if it will improve readablity — i.e. `src_mac` or
   `src_mac_addr` instead of `source_media_access_control_address`
+* Repeated fileds which cannot be packed into a sequence should
+  have `id`s containing a number in the end. Numbers may have a
+  visually-obvious structure, like "the first digit is major, the second
+  one is minor".
 * For fields that are designed to be used to detect file type (AKA
-  "magic values"), use `magic` name, or, if there are several of them,
-  `magic1`, `magic2`, etc.
+  "magic values"), use `signature` or `magic` name, or, if there are
+  several of them, like `signature` or `magic0`, `magic1`, etc.
```

Comment:
Ok, so, basically:

1. You specify "`signature` or `magic`", and don't specify any way to choose between these two. Given that its essentially the same stuff, I see no point in separation this into two names, and adding headache for ksy developers to choose between two proposed alternatives.
2. You want every numbered thing to start counting from 0. Any reasons to do so, i.e. any major examples of someone doing that?

Reply:
1 at KSY author's disposal. I personally prefer "signature" because it clearly states that this field is used for identification.
2 Just taste. If we start arrays from index 0 in the most of languages why not to follow the same convention here?